README.md

```diff
@@ -1,6 +1,54 @@
 # BMX
 
-BMX grants you API access to your AWS accounts, based on Okta credentials that you already own.
-It uses your Okta identity to create short-term AWS STS tokens, as an alternative to long-term IAM access keys.
+BMX provides API access to your AWS accounts using existing Okta credentials. Instead of long-term IAM access keys, BMX creates short-term AWS STS tokens using your Okta identity.
 
-This is a work in progress dotnet 7 branch
+# Installation
+
+Download the appropriate binary from the [releases](https://github.com/Brightspace/bmx/releases) page.
+
+# Commands
+
+## configure
+
+Set up the global BMX configuration file with the following command:
+```PowerShell
+bmx configure --org your-okta-organization --user your-okta-username
+```
+BMX will default to use the provided values. Okta account sessions are also saved when a configuration file is present. As such, it is not recommended to run `bmx configure` or create a configuration file on a machine with multiple users.
+
+Note: The flags provided are optional. Use `bmx configure -h` to view all available options.
+## print
+
+To setup AWS credentials in PowerShell, use:
+```PowerShell
+bmx print --account account-name --role role-name | iex
+```
+To setup AWS credentials in Bash, use:
+```Bash
+$(bmx print --account account-name --role role-name)
+```
+Note: The flags provided are optional. Use `bmx print -h` to view all available options.
+
+## write
+
+Create a new AWS credentials profile with the following command:
+```Powershell
+bmx write --account account-name --role role-name --profile my-profile
+```
+You can use your created profile by providing the `profile` flag in your AWS CLI commands, for example:
```

Comment:
The "`profile` flag" is very specific to the AWS CLI, but there are many other use cases that don't involve the AWS CLI. It's good to show this as an example, but I don't want to give people the impression that this is the only way to use profiles created by bmx.

Maybe this is better?
```suggestion
You can use your created profile by configuring any supporting AWS client. For example, for the AWS CLI :
```

Reply:
Yea that makes it clearer